internal/server/data/data.go

```diff
@@ -106,9 +106,7 @@ func initialize(db *DB) error {
 	}
 
 	db.DefaultOrg = org
-
-	db.Statement.Context = WithOrg(db.Statement.Context, org)
-	db.DefaultOrgSettings, err = GetSettings(db.DB)
+	db.DefaultOrgSettings, err = getSettingsForOrg(db.DB, org.ID)
```

Comment:
```suggestion
	db.OrgSettings, err = getSettingsForOrg(db.DB, org.ID)
```

This is a bit of a confusingly named field because these will no longer be the default settings once the default org modifies them

Reply:
Ah, this is supposed to mean "the settings for the default org", not "the default settings". 

I could create a new struct , or a `Settings *models.Settings` field on `models.Organization` to store this, which would make this more obvious.

Reply:
I added a comment for now, hoping that helps.  This is only used for setting telemetry so it's possible we'll remove this soon.